### ISSUE TITLE
Uncaught (in promise) TypeError: fs.writeSync is not a function

### ISSUE THREAD

**Issue by happybeing** [aaronpowell/webpack-golang-wasm-async-loader#16]
I'm trying to get the plugin working with go version go1.13.8 linux/amd64

I have the plugin and both examples building. Running the node example doesn't cause an error, but the result printed is '0'. In the web example, the browser console has the following error on loading:
```
Uncaught (in promise) TypeError: fs.writeSync is not a function
    wasmWrite wasm_exec.js:248
    run wasm_exec.js:475
    init gobridge.ts:28
    default_1 gobridge.ts:32
    go main.go:2
```
To get to this point I had to update the `./lib/wasm_exec.js` for v1.13 (see attached) but I'm not sure how to fix this error yet.
[wasm_exec.js.txt](https://github.com/aaronpowell/webpack-golang-wasm-async-loader/files/5466801/wasm_exec.js.txt)

**Comment by happybeing** [comment]
Minor update. I was correct that the node example was not working, but it was a separate issue. The node example works if I change `add()` in ./example/node/src/main.go to:
```
func add(this js.Value, args []js.Value) (interface{}, error) {
	ret := 0

	for _, item := range args {
		// This fails because item.String() is "<number: 1>" not "1"
		// val, err := strconv.Atoi(item.String())
		// if err != nil {
		// 	return 0, err
		// }

		val := item.Int()
		ret += val
	}

	return ret, nil
}
```
I found a similar issue ("syscall/js: make wasm_exec.js compatible with Webpack" [here](https://go-review.googlesource.com/c/go/+/208600/6/misc/wasm/wasm_exec.js#30)) related to `fs.writeSync()` but applying the fix for that does not fix the web example problem in the OP. Nor is this fix needed for the node example to work.

**Comment by happybeing** [comment]
I found a partial fix, at least it avoids the above error which occurs when executing `println("Web Assembly is ready")` in web/src/main.go and this is printed to the browser console when the page loads. The fix is another edit to `wasm_exec.js` to change the test on `global.fs` to detect the case where it is an object with an empty keys array, so that it creates an implementation of `fs.writeSync()` etc. So at line 37-ish I changed:
```
if (!global.fs) {
```
To:
```
if (!global.fs || Object.keys(global.fs) === 0 || Object.keys(global.fs).length === 0) {
```

The page displays ok, and clicking on "simulate an error" works, but changing integer values does not affect the "Value is now" value, which is always '0'.
